mkosi.postinst

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -e
+
+mkdir -p /root/build
+echo "directory /root/build" > ~/.gdbinit
```

Comment:
This assumes that the mkosi image will only be used for systemd debugging. It seems a bit presumptive, but is probably correct almost always.